addon/components/item-picker/feature-service-preview/template.hbs

```diff
@@ -1,10 +1,10 @@
 <h4>{{model.item.title}}</h4>
-<span class="shared-by-owner">{{t (concat _i18nScope "sharedBy") }}: {{model.item.owner}}</span>
 
 <div class="item-picker-current-item-preview-meta">
-  <div><i class="glyphicon icon-layers"></i>{{layerList.length}}</div>
-  <div><i class="glyphicon glyphicon-calendar"></i>{{format-time model.item.modified}}</div>
-  <div><i class="glyphicon glyphicon-file"></i>{{itemType}}</div>
+  <span class="shared-by-owner">{{t (concat _i18nScope "sharedBy") }}: {{model.item.owner}}</span>
+  <span><i class="glyphicon icon-layers"></i>{{layerList.length}}</span>
+  <span><i class="glyphicon glyphicon-calendar"></i>{{format-time model.item.modified}}</span>
+  <span><i class="glyphicon glyphicon-file"></i>{{itemType}}</span>
 </div>
```

Comment:
I think it was better with these just being divs and the container not a flex container. I don't see what is gained by this.

Reply:
With flexbox, can I take them out of the container or should I leave them in?

Reply:
You would need to leave them in the container because you want the container that contains these items to be a flex item, you don't want each of these elements to be a flex item. You are putting `justify-content: space-between` on the flex container so that applies to the flex items (the direct children of the flex container). You don't want the available space to be distributed between each of these. Let me know if this does not make sense and I can try to explain better.